### PR TITLE
Issue #11719: Activate all group for pitest-sizes

### PR DESCRIPTION
**Pull request by Vyom-Yadav**
#11719
Activating `ALL` group for `pitest-sizes`

Every mutator has been activated except `INLINE_CONSTS`

- Report with `ALL` group except `INLINE_CONSTS`: https://vyom-yadav.github.io/pitest-all-latest/pitest-sizes/index.html

### CODE REVIEW

File: .ci/pitest-suppressions/pitest-sizes-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  </mutation>
+
+  <mutation unstable="false">
+    <sourceFile>MethodLengthCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.sizes.MethodLengthCheck</mutatedClass>
+    <mutatedMethod>countUsedLines</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.NonVoidMethodCallMutator</mutator>
+    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getLineNo</description>
+    <lineContent>final int startLineNo = ast.getLineNo();</lineContent>
+  </mutation>
```
Comment (nick-mancuso) [inline .ci/pitest-suppressions/pitest-sizes-suppressions.xml:37]: @Vyom-Yadav please open PR with hardcoded mutation for this.

**Review by nick-mancuso — CHANGES_REQUESTED** [review]
Item:

File: .ci/pitest-suppressions/pitest-sizes-suppressions.xml (hunk 1 above)
Comment (Vyom-Yadav) [reply .ci/pitest-suppressions/pitest-sizes-suppressions.xml:37]: #11998, I won't be adding comment explanation comment at #11719, this particular mutant is very well documented at https://pitest.org/quickstart/mutators/#NON_VOID_METHOD_CALLS

**Review by nick-mancuso — APPROVED** [review]

**Comment by nick-mancuso** [comment]
CI failure in https://checkstyle.semaphoreci.com/workflows/66b81ad5-8e6a-47b7-80bf-95b24093f2d5?pipeline_id=2c4f223d-0f86-49fc-bff5-fd2290837d07 is not related to changes here.

**Pull request merged**